_config.yml

```diff
@@ -0,0 +1,15 @@
+# Setup
+title:            CoMPAS Architecture
+tagline:          'Architecture of the CoMPAS project'
+
+paginate:       5
+
+# Custom vars
+version:          0.0.1
```

Comment:
Is this the version of the website? 0.1.0 is a good start of the published version.

Reply:
It's a custom variable, it is shown on the Github Pages website itself.
I think it's good to have some kind of version shown on the website if it's actively being used.

On the other hand, you also have the complete history in the repo itself. So maybe the added value of an extra version is not that much (and it takes some extra effort in maintaining it).

Reply:
If it is not shown in on the github pages, 0.0.1 is fine for me.